src/db/pets/boar.ts

```diff
@@ -0,0 +1,37 @@
+import { Ability, Trigger, Pet } from "..";
+import { getPetIdentifiers } from "../database";
+
+function boarAbility(level: number): Ability {
+  return {
+    description: `Before attack: Gain +${level * 2}/+${level * 2}.`,
+    trigger: Trigger.BeforeAttack,
+    triggeredBy: {
+      kind: "Self",
+    },
+    effect: {
+      kind: "ModifyStats",
+      target: {
+        kind: "Self",
+      },
+      attackAmount: level * 2,
+      healthAmount: level * 2,
+      untilEndOfBattle: false,
+    },
+  };
+}
+
+export const boar: Pet = {
+  ...getPetIdentifiers("Boar"),
+  image: {
+    source: "noto-emoji",
+    commit: "e022fd6573782431ac9a65b520376b57511c31cd",
```

Comment:
https://github.com/googlefonts/noto-emoji/blob/f2a4f72bffe0212c72949a22698be235269bfab5/svg/emoji_u1f417.svg

I think `f2a4f72bffe0212c72949a22698be235269bfab5` or `9da2857552dd88c9826a90deaf034bb7bc3eb029` should be the `commit` here.

Reply:
Could you explain what these commits are? I simply copied that from the hippo file and left it alone since I saw another animal had the same commit.

Reply:
I think I see what the commit is supposed to be but when I made the change the icon is still just the boars head. Any advise on how to get the icon you linked?

Reply:
So the icons change over time in that repository, I guess Google re-does the design of them. For the boar: the current icon design in noto-emoji doesn't match whats in the game, instead we need one from that commit in 2017.

The code that downloads those emoji is in `src/web/ssr/writeEmojiAssets.ts` - I think it will avoid downloading an icon if you already have one on disk. To get a new one, you should be able to delete the `docs/assets` folder and run the build again 👍